cmd/deptool/analyze.go

```diff
@@ -252,7 +252,8 @@ func findLatestVersion(repo *git.Repository, latestCommitRef *plumbing.Reference
 	return commits[len(commits)-1], versions[len(commits)-1], workspaceVer[len(commits)-1], nil
 }
 
-func findCargoVersionForCommit(repo *git.Repository, pkgName string, commit *object.Commit) (string, bool, error) {
+//lint:ignore funlen gocyclo
```

Comment:
Isn’t it worth fixing the cyclomatic complexity? The function seems rather complex.

Reply:
yes, I agree with that. I was hoping to try and fix as many of the issues, without any substantial code change ( since that would require more testing, etc. ).
I'll refactor that in a subsequent PR, as this isn't really a concrete concern.